### PR TITLE
Issue Resolved: Fix Breaking-link Punctuation at end of URL in Comments #10947

### PR DESCRIPTION
**Pull request by rajvidg**
The commits made on 9th April, about updating two files are the required changes.

### CODE REVIEW

**Comment by TildaDares** [comment]
@rajvidg These changes are not needed for the issue you're resolving.

**Comment by rajvidg** [comment]
> @rajvidg These changes are not needed for the issue you're resolving.

Actually there are four changes and the last two are were the required ones. @TildaDares

File: README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 PublicLab.org
 ======
+I forked and cloned this repo.
```
Comment (TildaDares) [inline README.md:3]: @rajvidg Can you remove this file from the PR? Thanks!!

**Review by TildaDares — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (rajvidg) [reply README.md:3]: > @rajvidg Can you remove this file from the PR? Thanks!!

Yes I am trying to, watching some tutorials. :'(

Comment (rajvidg) [reply README.md]: > @rajvidg Can you remove this file from the PR? Thanks!!

I removed the changes and reverted the file to its original one, but it got created as another commit, but the files changes section contains the required files, I tried doing it using git commands but the README file was not showing up in the branch where I committed rest of the changes, can you please look into it. @TildaDares

Comment (TildaDares) [reply README.md]: @rajvidg You've resolved the issue 🎉

**Comment by TildaDares** [comment]
@rajvidg Can you add the issue number to the description above? Thanks!!

File: README.md (hunk 1 above)
Comment (rajvidg) [reply README.md]: > @rajvidg You've resolved the issue 🎉

Oh! thank you @TildaDares :))

**Comment by rajvidg** [comment]
> @rajvidg Can you add the issue number to the description above? Thanks!!

Ya sure!!

**Comment by TildaDares** [comment]
Also can you change the PR title to one that is more descriptive of the issue you're resolving?

**Comment by rajvidg** [comment]
> Also can you change the PR title to one that is more descriptive of the issue you're resolving?

Okay

**Comment by rajvidg** [comment]
The issue was: Fix Breaking-link Punctuation at end of URL in Comments, with the tag: #10947

**Review by TildaDares — APPROVED** [review]
Thanks for working on this @rajvidg!!

**Comment by rajvidg** [comment]
> Thanks for working on this @rajvidg!!

Thank you for giving me the chance :)) @TildaDares

**Review by RuthNjeri — APPROVED** [review]
Looks good @rajvidg!

**Comment by jywarren** [comment]
Thanks, everyone!!

**Comment by jywarren** [comment]
@TildaDares would you mind checking this in https://stable.publiclab.org once it publishes? Thank you!!

**Pull request merged**